src/deadline_worker_agent/boto/config.py

```diff
@@ -3,22 +3,32 @@
 from botocore.config import Config
 
 from .._version import __version__ as worker_agent_version
+from deadline.client import version as deadline_client_lib_version
+from openjd.sessions import version as openjd_sessions_version
+
+
+def construct_user_agent() -> str:
+    """Compute the user agent string to send over boto requests.
+    - Contains Deadline Worker Agent, Deadline Client Library, OpenJD Sessions.
+    """
```

Comment:
nit
- newline between `"""` and the comment?
- contains the versions of those packages?